src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+
```

Comment:
let' remove all empty lines from this method

Reply:
done